tests/admin/test_forms.py

```diff
@@ -19,29 +20,51 @@
 from enterprise.admin.forms import (EnterpriseCustomerAdminForm, EnterpriseCustomerIdentityProviderAdminForm,
                                     ManageLearnersForm)
 from enterprise.admin.utils import ValidationMessages
-from enterprise.course_catalog_api import NotConnectedToOpenEdX, get_all_catalogs
-from test_utils import fake_enrollment_api
+from enterprise.course_catalog_api import CourseCatalogApiClient
+from test_utils import fake_catalog_api, fake_enrollment_api
 from test_utils.factories import (EnterpriseCustomerFactory, EnterpriseCustomerUserFactory,
                                   PendingEnterpriseCustomerUserFactory, SiteFactory, UserFactory)
+from test_utils.fake_catalog_api import FAKE_PROGRAM_RESPONSE1, FAKE_PROGRAM_RESPONSE2
 
 FAKER = FakerFactory.create()
 
 
+class TestWithCourseCatalogApiMixin(object):
```

Comment:
To make this a proper mixin, let's do `self.catalog_api = self._make_catalog_api()` in this class's `setUp` method. Or, even inline that code into this class's `setUp`.